frontend/src/core/comments/components/CommentsList.css

```diff
@@ -1,12 +1,19 @@
 .comments-list {
     line-height: 22px;
-    padding: 8px;
 }
 
 .comments-list .comment {
     display: flex;
     margin-left: 58px;
 }
+.comments-list ul li {
+    border-bottom: 1px solid #5e6475;
```

Comment:
Only the last child among the pinned comments should have a border.

Reply:
I'm not too sure what to do here. Do you want the pinned comments to have no borders besides the last pinned comment?

Reply:
Correct. The apperance should be the same as on master. The only change would be the border between pinned comments and all comments.